File: c/use-case.c
#include <stdio.h>
#include "novacrypt.h"

int main()
{
    NovaCrypt nc;
    NovaCrypt_init(&nc, 9851, 987);

    char content[] = "Hello, World!";
    char encrypted[256];
    char decrypted[256];

    encrypt(&nc, content, encrypted);
    printf("C: Encrypted: %s\n", encrypted);

    decrypt(&nc, encrypted, decrypted);
    printf("C: Decrypted: %s\n", decrypted);

    return 0;
}

File: python/use-case.py
from novacrypt import NovaCrypt


nova_crypt = NovaCrypt(12345, 0)
encrypted = nova_crypt.encrypt('Hello, World!')
print('Python: Encrypted:', encrypted)
decrypted = nova_crypt.decrypt(encrypted)
print('Python: Decrypted:', decrypted)
